src/react.js

```diff
@@ -0,0 +1,20 @@
+/* @flow strict-local */
```

Comment:
I'd been following the pattern of `src/react-redux.js`, `src/react-native-safe-area-context.js`, and `src/react-navigation.js` (possibly others?). These have so far been for "type wrappers"; things like

```
/**
 * Exactly like the `useSelector` in `react-redux` upstream, but more typed.
 * […]
```

But

1. Maybe that's not the best place for those?
2. This one isn't a type wrapper; it's not modeled on an existing thing, it's a new thing.

So, putting it in `utils` sounds good.

Reply:
I've generally avoided putting new things in `src/utils/`, because I tend to feel that a "utils" directory makes things harder to find more than it makes them easier. It makes the list of things in the parent directory shorter; but now when you want to browse for something, the first thing you have to guess is whether it got called a "utility". Better for the top-level organization to be by what general area or subsystem the code belongs to.

If you take a look in the server/webapp repo, you'll see there are no directories with "util" names there, either. But there is this pattern, which gives me an idea here:
```
$ git ls-files | grep util
static/js/hash_util.js
static/js/keydown_util.ts
static/js/list_util.js
static/js/message_util.js
…
```

So for example there are a lot of files related to messages; and one of them, for some miscellaneous functions with minimal dependencies, is named `message_util.js`. That feels a lot better to me than a `util/messages.js` would, because you can still navigate it by starting with "the thing I'm looking for would be about messages".

Here, how about `src/reactUtils.js`?